cluster-management/debugging/manual-rollbacks/index.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: docs
+slug: guides
+title: Manual CoreOS Rollbacks
+category: cluster_management
+sub_category: debugging
+weight: 5
+---
+
+# Performing Manual CoreOS Rollbacks
+
+As much as we all love flawless, automatic updates, there may be occasions when
```

Comment:
frankly I don't care for the casual/conversational tone (feels a bit patronising), but who am I?